spec/acceptance/sources_spec.rb

```diff
@@ -66,6 +99,7 @@
           assert_match(/bob/, get_xml_value("//sources/source[@id='test']/@user", result.output).to_s, 'User did not match')
           assert_match(/.+/, get_xml_value("//sources/source[@id='test']/@password", result.output).to_s, 'Password was not saved')
           assert_match(/false/, get_xml_value("//sources/source[@id='test']/@disabled", result.output).to_s, 'Disabled did not match')
+          assert_match(/true/, get_xml_value("//sources/source[@id='test']/@bypassProxy", result.output).to_s, 'Bypass Proxy did not match')
         end
```

Comment:
The more recent idiom is to use rspec's `expect(some_var).to match(/true/)` although with all the other `assert_match`s continuing in that way is fine.